contracts/distribution/AlphaReleaseRuleSelector.sol

```diff
@@ -85,14 +95,12 @@ contract AlphaReleaseRuleSelector is Ownable, IAlphaReleaseRuleSelector {
     view
     returns (IAlphaReceiver[] memory, uint256[] memory)
   {
-    IAlphaReceiver[] memory receivers = new IAlphaReceiver[](ruleCount);
-    uint256[] memory amounts = new uint256[](ruleCount);
-    address currentReceivers = receiverList[HEAD];
-    for (uint256 i = 0; i < ruleCount; i++) {
-      receivers[i] = IAlphaReceiver(currentReceivers);
-      IAlphaReleaseRule releaseRule = rules[currentReceivers];
+    IAlphaReceiver[] memory receivers = new IAlphaReceiver[](receiverList.length);
+    uint256[] memory amounts = new uint256[](receiverList.length);
```

Comment:
Use receiverCount  instead of receiverList.length? (lines 98,99,100)

Reply:
Fixed